zerver/models.py

```diff
@@ -3313,6 +3313,23 @@ def increment_times_used(self) -> None:
         self.save(update_fields=["times_used"])
 
 
+class MissedMessageEmailEntry(models.Model):
+    """
+    Store the missedmessage events for a user which are yet to be processed.
+    """
+
+    user_profile: UserProfile = models.ForeignKey(UserProfile, on_delete=CASCADE)
+
+    message: Message = models.ForeignKey(Message, on_delete=CASCADE)
+    trigger: str = models.TextField(max_length=20)
```

Comment:
Do we need a `max_length` on this?  That's only required for `CharField`, and I don't think there's a reason to have one for this.

Reply:
I changed this to use `choices`. There's only 4 string values this field can.